docs/commands.md

```diff
@@ -35,34 +35,35 @@ commands:
     parameters:
       to:
         type: string
+        default: "Hello World"
     steps:
       - run: echo << parameters.to >>
 ```
 
 When invoking a command, the steps of that command are inserted where it's invoked. Commands can only be invoked as part of the sequence under `steps` in a job.
 
 ### Invoking other commands in your command
-Commands can use other commands in the scope of execution. For instance, if a command is declared locally inside your `.circleci` folder it can invoke other commands also declared locally. If a command is declared inside an Orb it can use other commands in that orb. We do not yet allow calling commands outside the scope of declaration.
+Commands can use other commands in the scope of execution. For instance, if a command is declared is declared inside your Orb it can use other commands in that orb. It can also use commands defined in other orbs that you have imported (e.g. `some-orb/some-command`).
```

Comment:
I feel like this could become it's own section on scope of commands, etc.

**nit**, "if a command is declared is declared..".